### PR TITLE
Unsupported CustomVisuals for Email Subscriptions 

### PR DESCRIPTION
**Pull request by ksgaipal**

### CODE REVIEW

**Comment by Court72** [comment]
@mihart

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by mihart** [comment]
#sign-off

File: powerbi-docs/collaborate-share/service-troubleshoot-subscribe.yml [hunk 1]
```diff
@@ -160,7 +160,7 @@ sections:
         answer: |
           There are certain Power BI features that will not work when using the subscription feature.
 
-          - Email subscriptions do not support most [custom visuals](../developer/visuals/develop-power-bi-visuals.md).  The exception is those Power BI custom visuals that have been [certified](../developer/visuals/power-bi-custom-visuals-certified.md).
+          - Email subscriptions do not support R, Python, Paginated Report Visuals, Power Apps, Power Automate, Visio and most [custom visuals](../developer/visuals/develop-power-bi-visuals.md). The exception is those Power BI custom visuals that have been [certified](../developer/visuals/power-bi-custom-visuals-certified.md).
```
Comment (LoboVerga) [inline powerbi-docs/collaborate-share/service-troubleshoot-subscribe.yml:163]:
```suggestion
          - Email subscriptions do not support R, Python, Paginated Report Visuals, Power Apps, Power Automate, Visio and most [custom visuals](../developer/visuals/develop-power-bi-visuals.md). The exception is those Power BI custom visuals that have been [certified](../developer/visuals/power-bi-custom-visuals-certified.md).
```

File: powerbi-docs/collaborate-share/service-troubleshoot-subscribe.yml [hunk 2]
```diff
@@ -160,7 +160,7 @@ sections:
         answer: |
           There are certain Power BI features that will not work when using the subscription feature.
```
Comment (LoboVerga) [inline powerbi-docs/collaborate-share/service-troubleshoot-subscribe.yml:161]:
```suggestion
           There are certain Power BI features that will not work when using the subscription feature.
```

File: powerbi-docs/collaborate-share/service-troubleshoot-subscribe.yml [hunk 3]
```diff
@@ -160,7 +160,7 @@ sections:
...
 
-          - Email subscriptions do not support most [custom visuals](../developer/visuals/develop-power-bi-visuals.md).  The exception is those Power BI custom visuals that have been [certified](../developer/visuals/power-bi-custom-visuals-certified.md).
```
Comment (LoboVerga) [inline powerbi-docs/collaborate-share/service-troubleshoot-subscribe.yml:163]: powerbi-docs/collaborate-share/service-troubleshoot-subscribe.yml

Comment (LoboVerga) [inline powerbi-docs/collaborate-share/service-troubleshoot-subscribe.yml:163]: -

**Review by LoboVerga — APPROVED** [review]

**Review by LoboVerga — APPROVED** [review]

**Pull request merged**